Apps/Sandcastle/gallery/Particle System Tails.html

```diff
@@ -0,0 +1,260 @@
+function getImage() {
+    if (!Cesium.defined(particleCanvas)) {
+        particleCanvas = document.createElement('canvas');
+        particleCanvas.width = 20;
+        particleCanvas.height = 20;
+        var context2D = particleCanvas.getContext('2d');
+        context2D.beginPath();
+        context2D.arc(8, 8, 8, 0, Cesium.Math.TWO_PI, true);
+        context2D.closePath();
+        context2D.fillStyle = 'rgb(255, 255, 255)';
+        context2D.fill();
+    }
+    return particleCanvas;
+}
+
+// Cesium Viewer Information
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    shouldAnimate : true
+});
+
+viewer.scene.debugShowFramesPerSecond = true;
+
+// particle system information
+var modelMatrix = Cesium.Transforms.eastNorthUpToFixedFrame(Cesium.Cartesian3.fromDegrees(-75.59777, 40.03883));
+
+var centerLocation = new Cesium.Cartesian3(0, 0, 1);
+var cartographicCenterLocation = Cesium.Cartographic.fromCartesian(centerLocation);
+var cartographicIncrease = 800.0;
+cartographicCenterLocation.height += cartographicIncrease;
+centerLocation = Cesium.Cartographic.toCartesian(cartographicCenterLocation);
+
+// Add plane to scene
+var planeOffsetFromParticles = 50;
+var planePosition = Cesium.Cartesian3.fromDegrees(-75.59777, 40.03883, cartographicIncrease + planeOffsetFromParticles);
+var hpr = new Cesium.HeadingPitchRoll(0.0, Cesium.Math.PI_OVER_TWO, 0.0);
+var orientation = Cesium.Transforms.headingPitchRollQuaternion(planePosition, hpr);
+var entity = viewer.entities.add({
+    //Load the Cesium plane model to represent the entity
+    model : {
+        uri : '../../SampleData/models/CesiumAir/Cesium_Air.gltf',
+        scale : 3.5
+    },
+    position : planePosition,
+    orientation : orientation
+});
+viewer.trackedEntity = entity;
+
+// options
+var rocketOptions = {
+    numberOfParticles : 50.0,
+    iterOffset :  0.1,
+    cartographicStep : 0.000001,
+    baseRadius : 0.0005,
+
+    colorOptions : [{
+        minimumRed : 1.0,
+        green : 0.5,
+        minimumBlue : 0.05,
+        alpha : 1.0
+    }, {
+        red : 0.9,
+        minimumGreen : 0.6,
+        minimumBlue : 0.01,
+        alpha : 1.0
+    }, {
+        red : 0.8,
+        green : 0.05,
+        minimumBlue : 0.09,
+        alpha : 1.0
+    }, {
+        minimumRed : 1,
+        minimumGreen : 0.05,
+        blue : 0.09,
+        alpha : 1.0
+    }],
+
+    particleSystems : []
+};
+
+var cometOptions = {
+    numberOfParticles : 100.0,
+    iterOffset : 0.003,
+    cartographicStep : 0.0000001,
+    baseRadius : 0.0005,
+
+    colorOptions : [{
+        red : 0.6,
+        green : 0.6,
+        blue : 0.6,
+        alpha : 1.0
+    }, {
+        red : 0.6,
+        green : 0.6,
+        blue : 0.9,
+        alpha : 0.9
+    }, {
+        red : 0.5,
+        green : 0.5,
+        blue : 0.7,
+        alpha : 0.5
+    }, {
+        red : 0.5,
+        green : 0.5,
+        blue : 0.5,
+        alpha : 0.9
+    }],
+
+    particleSystems : []
```

Comment:
I think it may be a little confusing to the reader that these are saved in `options`. Can we store these arrays as their own variables?

Reply:
the only reason i do this is because inside the method, there wouldn't be a way of knowing which options was passed, so i'd need to pass another object through and there were already a lot - i think i can streamline the code a bit more tho so this passing another obj wouldnt be that bad now